### PR TITLE
Upgrade legacy learners that paid and are enrolled, have exam attempt

### PR DESCRIPTION
**Pull request by annagav**
#### What are the relevant tickets?
Fix https://github.com/mitodl/mitxonline/issues/960

#### What's this PR do?
This is a management command that take in a file and upgrades learners in their current enrollment

#### How should this be manually tested?
Create a csv file that has data of a learner in your local mitxonline environment, that is auditing a a current course.
In the csv file add a record: <learner_email>, <course_run_key>
Run the command `./manage.py upgrade_eligible_users input_file.csv`
The enrollment should get upgrades to 'verified'.

### CODE REVIEW

File: courses/management/commands/upgrade_eligible_users.py [hunk 1]
```diff
@@ -0,0 +1,85 @@
...
+
+                if course_run is None:
+                    self.stderr.write(
+                        self.style.ERROR(
+                            f"Can't find courserun for {row[1]}, skipping row"
+                        )
+                    )
+                    continue
+                upgrade_count = 0
+                enrollment = CourseRunEnrollment.objects.filter(
```
Comment (jkachel) [inline courses/management/commands/upgrade_eligible_users.py:62]: missing the fetch part - this returns a QuerySet

**Review by jkachel — CHANGES_REQUESTED** [review]
Algorithm looks good and `create_edx_enrollments` is the right way to go about this (since that'll also perform the upgrade in edX); there's a simple error that needs to be resolved but otherwise is OK.

**Review by jkachel — CHANGES_REQUESTED** [review]
One more change regarding handling failed enrollments

**Comment by annagav** [comment]
@jkachel I added a check for edx_request_success to log out failed enrollments. Thanks

**Review by jkachel — APPROVED** [review]
LGTM - ready to merge once tests pass (looks like it needs a black run) 👍

**Pull request merged**